gateway/nginx.conf.prod

```diff
@@ -0,0 +1,46 @@
+server {
+    listen 80;
+    server_name app.formulai.cloud;
+
+    location /.well-known/acme-challenge/ {
+        root /var/www/certbot;
+    }
+
+    location / {
+        return 301 https://$host$request_uri;
+    }
+}
+
+server {
+    listen 443 ssl;
+    server_name app.formulai.cloud;
+
+    ssl_certificate /etc/letsencrypt/live/app.formulai.cloud/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/app.formulai.cloud/privkey.pem;
+
+    ssl_protocols TLSv1.2 TLSv1.3;
+    ssl_prefer_server_ciphers on;
+
+    location /api/survey {
```

Comment:
Perhaps we could use https://nginx.org/en/docs/ngx_core_module.html#include at one point to not have to redefine same paths in both files, but it's something for the future.